src/index.js

```diff
@@ -88,7 +92,7 @@ import VerticalTab from '../components/tab/VerticalTab.vue'
 import WorkspaceTab from '../components/tab/WorkspaceTab.vue'
 
```

Comment:
This has been missed.

Reply:
Oh I left that extra `.` since I saw you address it in your upgrade PR 😆

Reply:
I always replace `..` with `.` otherwise I can't build the library :joy:.

Reply:
Removed them all ✅